Description: Rename to avoid file conflict with audiofile-tools
 Note that upstream wishes to switch to a different name
Author: mirabilos <tg@debian.org>
Forwarded: part of https://github.com/musescore/sftools/pull/18

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,7 +22,7 @@ set(QT_USE_QTXML         TRUE)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
 
-add_executable(sfconvert
+add_executable(musescore-sfconvert
    sfconvert.cpp sfont.cpp xml.cpp
    )
 
@@ -97,7 +97,7 @@ else (MINGW)
       ${VORBIS_INCDIR}
    )
 
-   target_link_libraries(sfconvert
+   target_link_libraries(musescore-sfconvert
       ${QT_LIBRARIES}
       ${OGG_LIB}
       ${VORBIS_LIB}
@@ -105,5 +105,5 @@ else (MINGW)
       ${SNDFILE_LIB}
    )
 
-   install(TARGETS sfconvert RUNTIME DESTINATION bin)
+   install(TARGETS musescore-sfconvert RUNTIME DESTINATION bin)
 endif (MINGW)
